tests/utils/profilehandler/profile_handler.go

```diff
@@ -201,6 +204,12 @@ func GenerateClusterCreateFlags(profile *Profile, client *rosacli.Client) ([]str
 			clusterConfiguration.Aws.Sts.ControlPlaneRoleArn = accRoles.ControlPlaneRole
 
 		}
+
+		if profile.ClusterConfig.SharedVPC {
+			sharedVPCRolePrefix = accountRolePrefix
+			sharedVPCRoleArn = fmt.Sprintf("arn:aws:iam::641733028092:role/%s-shared-vpc-role", sharedVPCRolePrefix)
```

Comment:
Can this part get from backend? Hardcode with an AWS account ID is not good here. And it can not be run across env like prod/stage/int